docs/guides/modules/orchestrate/pages/set-up-triggers.adoc

```diff
@@ -0,0 +1,121 @@
+. Choose when top run the pipeline by selecting an option from the *Event* menu.
++
+****
+Using multiple pipelines and triggers, you can build different pipelines for different events, For example, you could have:
+
+- A `build-test` pipeline that runs on every push. Select "All pushes".
+- A `e2e-tests` pipeline, longer and more costly, that runs once when the PR is taken out of draft. Select "PR marked ready for review".
+- A `teardown-env` pipeline that runs when a PR is merged. Select "PR merged".
+
+For a full list, see the xref:github-trigger-event-options.adoc[GitHub trigger event options] page.
+****
+* The *Event Source* defaults to the repository associated with the pipeline.
++
+TIP: You can change the event source to a different repository if you wish. If you do change the event source so that it does not match the pipeline's config/checkout source an option will appear to specify which branch in the config/checkout source repository you wish to use for the trigger.
+. Select btn:[Save]
+
+To verify your trigger is set up correctly, trigger an event from your repository.
+--
+Custom webhook::
++
+--
+Setting up a custom webhook trigger allows you to trigger pipelines from external services. Any service that can emit a webhook or make a cURL request can be set up in this way to trigger a pipeline. For more information, see the xref:triggers-overview.adoc#trigger-a-pipeline-from-a-custom-webhook[Triggers overview] page.
+
+To add a custom webhook trigger, follow these steps:
+
+include::ROOT:partial$pipelines-and-triggers/custom-webhook-setup.adoc[]
+
+To verify your trigger is set up correctly, trigger an event from your third party system.
+--
+GitLab::
++
+--
+
+When a trigger is created, CircleCI registers a webhook with GitLab. Push events from GitLab are sent to CircleCI. CircleCI then uses the event data to determine _if_ a pipeline should run, and if so, _which_ pipeline should be run.
+
+In addition to a pipeline, each trigger includes the webhook URL, and in this scenario, a CircleCI-created GitLab token. The webhook URL and GitLab token are used to securely register the webhook within GitLab in order to receive push events from your GitLab repository. You can view webhooks for a project in GitLab at menu:Settings[Webhooks].
+
+image::guides:ROOT:add-trigger.png[Add a trigger]
+
+To add a trigger for a pipeline, follow these steps:
+
+. In the link:https://app.circleci.com/[CircleCI web app] select **Projects** in the sidebar
+. Find your project in the list, select the ellipsis image:guides:ROOT:icons/more.svg[more icon, role="no-border"] next to it and choose **Project Settings**.
+. Select **Triggers** in the sidebar.
++
+TIP: If you do not see **Triggers** in the sidebar, your project is integrated using either xref:integration:github-integration.adoc[GitHub OAuth] or xref:integration:bitbucket-integration.adoc[Bitbucket Cloud] and these steps do not apply.
```

Comment:
```suggestion
TIP: If you do not see **Triggers** in the sidebar, your project may not yet have any Gitlab pipelines. To create one, navigate to the **Project setup** tab and click the link to create a new Gitlab pipeline at the bottom of the page. 
If you do not see this link, your project may be integrated using either xref:integration:github-integration.adoc[GitHub OAuth] or xref:integration:bitbucket-integration.adoc[Bitbucket Cloud] and these steps do not apply. 
```

Reply:
@rosieyohannan if it's not clear let me know and I'll explain what I'm trying to say